lead/models.py

```diff
@@ -166,12 +167,6 @@ def get_for(user):
     def can_get(self, user):
```

Comment:
Not sure but `can_get` should probably be removed from here.

Reply:
Also remove old `get_for` method. And remove these from Entry and Assessment too.